firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -228,6 +228,17 @@ public InstallationTokenResult generateAuthToken(
         if (httpResponseCode == 200) {
           return readGenerateAuthTokenResponse(httpsURLConnection);
         }
+
+        if (httpResponseCode == 401) {
+          return TokenResult.builder()
+              .setResponseCode(TokenResult.ResponseCode.REFRESH_TOKEN_ERROR)
+              .build();
+        }
```

Comment:
So, the request was not authenticated, what is happening next?
Are we deleting the PersistedInstallationEntry?
If yes, where is this done?
Can you please add a comment what REFRES_TOKEN_ERROR will trigger elsewhere?